t/ui/23-audit-log.t

```diff
@@ -15,7 +15,7 @@ use OpenQA::Client;
 
 use OpenQA::SeleniumTest;
 
-OpenQA::Test::Case->new->init_data;
+OpenQA::Test::Case->new->init_data(fixtures_glob => '01-jobs.pl 04-products.pl');
```

Comment:
Now I realized that you added this which we should avoid to save runtime and reduce inter-dependencies of tests. Well, you were right, we should have discarded review approvals after changes. At least here it would have prevented that ;) I will try to fix that.